### ISSUE TITLE
Photo Editor not working

### ISSUE THREAD

**Issue by ZorisBube** [Ademking/BetterViewer#48]
Hi!
I've been using BetterViewer for a week. Thanks for this great addon. But Photo Editor is not working. When selected a blank  editor tab opens but nothing happens. I don't know if it's only for me but tried it on an different Firefox profile without other extensions. A blank one i mean.. I guess there is a problem with that feature.  
Best regards!

**Desktop**
 - OS: Windows 10
 - Browser : Firefox
 - Version : 108.0.1

**Comment by TheRealBix** [comment]
Same issue here, 

on Firefox 110.

**Comment by Dantasstic** [comment]
Yup, same here. I just installed this add-on based on a tech article I read, but it seems like it is no longer being updated for bug fixes? That's a shame

Desktop

    OS: Windows 10
    Browser : Firefox
    Version : 109.0.1

**Comment by Ademking** [comment]
The issue has been fixed in v2 of BetterViewer. Please update to the latest version and it should work fine now. Let me know if you run into any more problems!
